src/Features/Core/Portable/IncrementalCaches/SymbolTreeInfoIncrementalAnalyzerProvider.cs

```diff
@@ -42,7 +43,7 @@ private struct MetadataInfo
             /// Note: can be <code>null</code> if were unable to create a SymbolTreeInfo
             /// (for example, if the metadata was bogus and we couldn't read it in).
             /// </summary>
```

Comment:
we already documented that we knew this could be null.  But we didn't name things effectively to make it clear at use sites that that coul dbe hte case.  I audited all use sites appropriately.